Sources/SwiftSyntaxBuilder/Syntax/SyntaxListBuilder.swift

```diff
@@ -0,0 +1,20 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+@_functionBuilder
+public struct SyntaxListBuilder {
```

Comment:
This type and the buildBlock function could both be internal.

Reply:
Merged into a file in f2ef721, and removed public modifiers in fe3a245.